spec/system/dashboard/users_spec.rb

```diff
@@ -25,13 +24,22 @@
         other_user.id,
         href: dashboard_user_path(other_user)
       )
-
-      expect(page).to have_content("Email")
-      expect(page).to have_content(user.email)
-      expect(page).to have_content(other_user.email)
-      expect(page).to have_no_content(different_user.email)
-      expect(page).to have_content("Last sign in at")
-      expect(page).to have_content("Invitation accepted at")
+      expect(page).to have_link(
+        User.human_attribute_name(:email),
+        href: dashboard_users_path(sort: "email", direction: "asc")
+      )
+      expect(page).to have_link(
+        User.human_attribute_name(:invitation_accepted_at),
+        href: dashboard_users_path(sort: "invitation_accepted_at", direction: "asc")
+      )
+      expect(page).to have_link(
+        User.human_attribute_name(:last_sign_in_at),
+        href: dashboard_users_path(sort: "last_sign_in_at", direction: "asc")
+      )
+      expect(page).to have_link(
+        User.human_attribute_name(:created_at),
+        href: dashboard_users_path(sort: "created_at", direction: "asc")
+      )
```

Comment:
To me, I think it's a bit too much and has little values on repeat all assertions here for each page and column like this.

I would rather test have just one scenario to verify if I could be able to click see the default sort, then I I click sort in the opposite direction then I see the result that I expect.

Another test should only verify which column that could be sortable by assertion with class selector something like this.

```
expect(page).to have_sortable_column(:email)
expect(page).to have_sortable_column(:last_sign_in_at)

def have_sortable_column(column)
  have_link(".sortable.#{column}")
end
```